ui/src/views/ShareTerminal/index.vue

```diff
@@ -61,8 +60,6 @@ const warningIntervalId = ref<number>(0);
 
 const onlineUsersMap = reactive<{ [key: string]: any }>({});
 
-terminalStore.setTerminalConfig('enableZmodem', false);
-
 onUnmounted(() => {
   clearInterval(warningIntervalId.value);
 });
```

Comment:
There don't appear to be any obvious differences or irregularities with this code snippet. However, I recommend running it through your editor's linting features if you're working in an IDE. If no errors are found there, no further changes could improve the code.